core-bundle/contao/templates/twig/backend/component/message/_message.html.twig

```diff
@@ -0,0 +1,31 @@
+{% trans_default_domain "contao_default" %}
+
+{% block message_component %}
+    {% set success = success is defined ? success : true %}
+    {% set show_close_button = show_close_button is defined ? show_close_button : not success %}
+
+    {% set message_attributes = attrs()
+        .addClass('message')
+        .addClass('message--info', success)
+        .addClass('message--error', not success)
+        .set('aria-live', 'polite', success)
+        .set('aria-alert', 'true', not success)
+        .set('data-contao--message-outlet-target', 'message') %}
+    <div{{ message_attributes }}>
+        <div class="inner">
+            {% if show_close_button or contao.backend_user.doNotHideMessages|default(false) %}
+                {% set close_button_attributes = attrs()
+                    .addClass('close')
+                    .set('type', 'button')
+                    .set('data-action', 'contao--message-outlet#removeMessage')
+                    .set('aria-label', 'MSC.cancelBT'|trans)
+                    .mergeWith(close_button_attributes|default) %}
+                <button{{ close_button_attributes }}>x</button>
```

Comment:
Should we use a proper X icon here? https://lucide.dev/icons/x

Reply:
That would be better, I guess. But let's change that in all places then (in a follow-up PR): 
* messages
* the old info dialog
* the template studio info panel
* …

Reply:
See https://github.com/contao/contao/issues/8074